src/epochs.lua

```diff
@@ -87,27 +83,21 @@ EpochSettings = EpochSettings
 		maxObservers = 50,
 		epochZeroStartTimestamp = 1719900000000, -- July 9th, 00:00:00 UTC
 		durationMs = constants.defaultEpochDurationMs, -- 24 hours
-		distributionDelayMs = constants.distributionDelayMs, -- 40 minutes (~ 20 arweave blocks)
 	}
 
 --- @type Timestamp|nil
 NextEpochsPruneTimestamp = NextEpochsPruneTimestamp or 0
 
---- Gets a deep copy of all the epochs
---- @return table<number, Epoch> # A deep copy of the epochs indexed by their epoch index
-function epochs.getEpochs()
-	return utils.deepCopy(Epochs) or {}
-end
-
+--- TODO: consider removing this handler and prune function below, epochs should manage their own lifecycle
```

Comment:
Still relevant?

Reply:
yeah, we needed prune epochs for the initial prune, but should be able to remove it now

Reply:
removed in https://github.com/ar-io/ar-io-network-process/pull/353/commits/11b9e1c33bb0975520a93ec5193b074670982a3f